types/types.go

```diff
@@ -314,6 +315,11 @@ type Digest struct {
 	UsersFile string
 }
 
+// Forward authentication
+type Forward struct {
+	Address string
```

Comment:
I'm agree with @emilevauge for the TLS connection.
Indeed, I tried to test the PR by connecting Traefik to a [docker_auth](https://github.com/cesanta/docker_auth) server but this server allows only HTTPS connections...